.github/mergify.yml

```
@@ -21,6 +20,17 @@ pull_request_rules:
       - status-success=test-suite (nautilus)
       - status-success=test-suite (octopus)
       - status-success=test-suite (pacific)
+      - or:
+        - and:
+          - label!=API
+          - "#approved-reviews-by>=1"
+        - and:
+          - label=API
+          - "#approved-reviews-by>=2"
+        - and:
+          - label=API
+          - "#approved-reviews-by>=1"
+          - "updated-at<10 days ago"
```

```suggestion
      - or:
        - "#approved-reviews-by>=2"
        - and:
          - "#approved-reviews-by>=1"
          - or:
            - label!=API
            - "updated-at<10 days ago"
```
If you start the logic from the number of reviews, it gets a bit shorter.

---

The docs for mergify at https://docs.mergify.io/conditions/#combining-conditions-with-operators say `The depth on combined conditions is limited to 3.` but I wasn't clear where they started counting and thus I opted to go for a "flat" style. I started out with something like you have suggested but after worrying about the nesting and re-writing it in the flat style, I like the flat style.